FEAT :: Submitted 페이지 퍼블리싱

j02on opened the pull request:
isColor를 사용해 true false 값을 관리하여 CompleteContents의 이미지, 버튼 색상을 변경 하는 것을 관리하였습니다.

j02on committed:
FEAT :: Submitted 페이지 퍼블리싱

isColor를 사용해 true false 값을 관리하여 CompleteContents의 이미지, 버튼 색상을 변경 하는 것을 관리하였습니다.

j02on committed:
FEAT :: Submitted Router

라우터를 설정하였습니다.

dutexion commented on apps/internship/src/assets/complete/Check.tsx:
```diff
@@ -0,0 +1,41 @@
+interface isCheck {
+  isColor?: boolean;
```
user인가 admin인가에 따라서 색이 변경되는 것이기 때문에,
isColor보다 `isAdmin:boolean` 혹은 `userType: 'user' | 'admin'`으로 작성하는게 어울릴 것 같습니다

dutexion commented on apps/internship/src/assets/complete/Check.tsx:
icon또한 공통적으로 사용될 수 있기 때문에, 패키지로 분리하여 관리하는게 좋을 것 같습니다

j02on replied:
저 아이콘은 거의 공통적으로 사용되지 않는 아이콘이라 따로 빼지 않았습니다

dutexion commented on apps/internship/src/components/complete/CompleteContents.tsx:
```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+import { color } from "@entry/design-token";
+
+interface isComplete {
+  children?: string;
+  isColor?: boolean;
+}
```
다른 곳에서 상속 받지 않고 확장하지 않을 타입이라면
interface보다는 type alias가 어울릴 수 있습니다

dutexion replied:
또한 "is"는 boolean 타입을 네이밍 할 때 주로 사용합니다

dutexion commented on apps/internship/src/pages/Submitted.tsx:
```diff
@@ -0,0 +1,19 @@
+import { React } from "react";
+import styled from "@emotion/styled";
+
+export const Submitted = () => {
+  return (
+    <SubmittedContainer>
+      <CompleteContents children="제출이 완료 되었습니다!" isColor={true} />
```
children 파라미터를 사용할 때는
`<CompleteContents children="제출이 완료 되었습니다!"/>`보다는
`<CompleteContents>제출이 완료 되었습니다!</CompleteContents>`가 가독성이 좋을 수 있습니다

dutexion reviewed (commented):
페이지 혹은 컴포넌트 pr 날릴 때에는 빠른 협업을 위해 스크린샷을 첨부하는 것도 좋은 방법입니다!

j02on committed:
FIX :: Submitted 페이지 리뷰 수정

check.tsx의 isColor isAdmin로 네이밍 변경, CompleteContents.tsx의 interface를 type alias로 수정,
interface의 네이밍 수정,
children 사용법 수정을 하였습니다.
